ee/clickhouse/queries/actor_base_query.py

```diff
@@ -36,15 +36,12 @@ class SerializedGroup(TypedDict):
     properties: Dict[str, Any]
 
 
-SerializedActor = Union[SerializedPerson, SerializedGroup]
-Actor = Union[Person, Group]
-
-
 class ActorBaseQuery:
     aggregating_by_groups = False
+    entity: Optional[Entity] = None
 
     def __init__(self, team: Team, filter: Filter, entity: Optional[Entity] = None):
-        self.team = team
+        self._team = team
         self.entity = entity
         self.filter = filter
 
```

Comment:
for the line below: 

must return **query & params**, that when run return list of uuids?